app/routes/content.js

```diff
@@ -7,6 +7,9 @@ export default Ember.Route.extend(ResetScrollMixin, {
     },
     setupController(controller, model) {
         controller.set('activeFile', model.get('primaryFile'));
+        Ember.run.scheduleOnce('afterRender', this, function() {
+            MathJax.Hub.Queue(['Typeset', MathJax.Hub]);  // jshint ignore:line
```

Comment:
Insofar as all DOM elements should be ready, does `Ember.$` not work here? This is rendering on the entire document, not just the parts of the content page of interest.